app/src/main/java/com/duckduckgo/app/statistics/pixels/Pixel.kt

```diff
@@ -159,7 +160,13 @@ interface Pixel {
         FEEDBACK_NEGATIVE_SUBMISSION("mfbs_%s_%s_%s"),
 
         AUTOCOMPLETE_BOOKMARK_SELECTION("m_aut_s_b"),
-        AUTOCOMPLETE_SEARCH_SELECTION("m_aut_s_s")
+        AUTOCOMPLETE_SEARCH_SELECTION("m_aut_s_s"),
```

Comment:
Interesting, why not use the Asana task and the Experiment definition subtask there? There's a link to the Grafana dashboard and the cases when each of the pixels are fired.

Reply:
It's a good question. It makes it easier for the reviewer to review if they have solid test cases outlined here rather than hunting through various Asana tasks. +1 to posting a link to the Grafana dashboard as well though.

Reply:
@malmstein actually, that could be a good idea to discuss on our mobile meeting. I was thinking that creating an Asana task under each project describing WHEN exactly each pixel should be fired and after what actions, like we do in the PR description, if well explained could be used by the reviewer, and also for product people or other teammates to understand any data available.